console/console-init/ui/src/pages/ConnectionDetail/ConnectionLinksWithFilterAndPaginationPage.tsx

```diff
@@ -89,7 +89,7 @@ export const ConnectionLinksWithFilterAndPaginationPage: React.FunctionComponent
         size={"lg"}
         className={css(GridStylesForTableHeader.filter_left_margin)}
       >
-        Links
+        Links for connection - {connectionname}
```

Comment:
Naming convension is not being followed here for 'connectionname'. Can you please change that?